### ISSUE TITLE
Iterate and display Marks - Feature Request

### ISSUE THREAD

**Issue by zedan185** [XVimProject/XVim#986]
I have been using vim-signature for a while. It allows me to display all my Vim Marks , iterate through them one after the other , generate them , delete a mark. I would like a way to iterate through all my marks in Vim (whether global or local)

**Comment by 99arobe** [comment]
I agree, vim-signature is really nice. Such features would be useful with XVim. 

The closest we could get at the moment is simply `:marks`. This lists all of the marks that have been set and provides line numbers so that you could jump to them (e.g., `25gg` or `11G`). It's not exactly a way to smoothly iterate over the marks that are available but it's better than nothing I guess.
